### ISSUE TITLE
FlexCloud: Deregister failure

### ISSUE THREAD

**Issue by briandrawert** [StochSS/stochss#428]
I can't get my flex cloud workers to register or deregister.  When I click "Stop", the following error shows up in stderr.log
```
ERROR    2015-06-07 19:45:57,490 wsgi.py:278] 
Traceback (most recent call last):
  File "/Users/brian/Desktop/research/stochss.develop/sdk/python/google/appengine/runtime/wsgi.py", line 266, in Handle
    result = handler(dict(self._environ), self._StartResponse)
  File "/Users/brian/Desktop/research/stochss.develop/sdk/python/lib/webapp2-2.3/webapp2.py", line 1519, in __call__
    response = self._internal_error(e)
  File "/Users/brian/Desktop/research/stochss.develop/sdk/python/lib/webapp2-2.3/webapp2.py", line 1511, in __call__
    rv = self.handle_exception(request, response, e)
  File "/Users/brian/Desktop/research/stochss.develop/sdk/python/lib/webapp2-2.3/webapp2.py", line 1505, in __call__
    rv = self.router.dispatch(request, response)
  File "/Users/brian/Desktop/research/stochss.develop/sdk/python/lib/webapp2-2.3/webapp2.py", line 1253, in default_dispatcher
    return route.handler_adapter(request, response)
  File "/Users/brian/Desktop/research/stochss.develop/sdk/python/lib/webapp2-2.3/webapp2.py", line 1077, in __call__
    return handler.dispatch()
  File "/Users/brian/Desktop/research/stochss.develop/sdk/python/lib/webapp2-2.3/webapp2.py", line 547, in dispatch
    return self.handle_exception(e, self.app.debug)
  File "/Users/brian/Desktop/research/stochss.develop/sdk/python/lib/webapp2-2.3/webapp2.py", line 545, in dispatch
    return method(*args, **kwargs)
  File "/Users/brian/Desktop/research/stochss.develop/app/backend/backend_handler.py", line 877, in get
    self.__handle_prepare_vms_op()
  File "/Users/brian/Desktop/research/stochss.develop/app/backend/backend_handler.py", line 859, in __handle_prepare_vms_op
    FlexBackendWorker(infra_manager=infra, agent=agent, reservation_id=reservation_id).prepare_vms(parameters)
  File "/Users/brian/Desktop/research/stochss.develop/app/backend/backend_handler.py", line 113, in prepare_vms
    parameters=parameters)
  File "/Users/brian/Desktop/research/stochss.develop/app/backend/backend_handler.py", line 313, in __poll_instances_status
    self.agent.deregister_some_instances(parameters, terminate_ins_ids)
  File "/Users/brian/Desktop/research/stochss.develop/app/backend/agents/flex_agent.py", line 202, in deregister_some_instances
    parameters=parameters)
TypeError: __deregister_flex_vm() takes at least 6 arguments (5 given)
```

**Comment by dnath** [comment]
I missed this during refactoring. It should be okay now in 3de54402abd19e0e43b70657a581d48153ecd00a

**Comment by briandrawert** [comment]
Closed in #383
